js/client-chat.js

```diff
@@ -607,18 +607,35 @@
 			case 'rating':
 			case 'ladder':
 				if (!target) target = app.user.get('userid');
+
+				var targets = target.split(',');
+				var formatTargeting = false;
+				var formats = {};
+				for (var i = 1, len = targets.length; i < len; i++) {
+					formats[toId(targets[i])] = 1;
+					formatTargeting = true;
+				}
+
 				var self = this;
-				$.get(app.user.getActionPHP() + '?act=ladderget&user='+encodeURIComponent(target), Tools.safeJSON(function(data) {
+				$.get(app.user.getActionPHP() + '?act=ladderget&user='+encodeURIComponent(targets[0]), Tools.safeJSON(function(data) {
 					try {
 						var buffer = '<div class="ladder"><table>';
-						buffer += '<tr><td colspan="8">User: <strong>'+target+'</strong></td></tr>';
+						buffer += '<tr><td colspan="8">User: <strong>'+targets[0]+'</strong></td></tr>';
 						if (!data.length) {
 							buffer += '<tr><td colspan="8"><em>This user has not played any ladder games yet.</em></td></tr>';
 						} else {
 							buffer += '<tr><th>Format</th><th><abbr title="Elo rating">Elo</abbr></th><th><abbr title="user\'s percentage chance of winning a random battle (aka GLIXARE)">GXE</abbr></th><th><abbr title="Glicko-1 rating: rating±deviation">Glicko-1</abbr></th><th>COIL</th><th>W</th><th>L</th><th>T</th></tr>';
-							for (var i=0; i<data.length; i++) {
+							var hiddenFormats = [];
+							var formatLength = data.length;
+							for (var i=0; i<formatLength; i++) {
 								var row = data[i];
-								buffer += '<tr><td>'+row.formatid+'</td><td><strong>'+Math.round(row.acre)+'</strong></td><td>'+Math.round(row.gxe,1)+'</td><td>';
+								if (!formatTargeting || formats[row.formatid]) {
+									buffer += '<tr>';
+								} else {
+									buffer += '<tr class="hidden">';
```

Comment:
You probably shouldn't use the HTML sanitizer, then... There are a few workarounds.

If you can't figure out any other workaround, I'd accept replacing `.ladder .hidden {` with `.hidden {` in the CSS files.